css/css-backgrounds/background-332.html

```diff
@@ -22,7 +22,7 @@
 
         test(function() {
             assert_equals(cs.getPropertyValue("background-image"),
-                "url(support/60x60-green.png)", "background specified value for background-image");
+                'url("' + new URL("support/60x60-green.png", window.location.href).href + '")', "background specified value for background-image");
```

Comment:
While at it maybe worth changing "specified" by "computed"? This is totally not testing the specified value :)

Reply:
Yep, good catch. Really, that should be done for the entire test (and in all of the `background-33*.html` files), but I just changed the individual test description for now.